# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1552675520 0
#      Fri Mar 15 18:45:20 2019 +0000
# Node ID 7fdbb03d5f76bd0e583d34df587b6abe6fbd4901
# Parent  ab9520ef389558ee1e2513efae8e17f58196cda0
include -Woverloaded-virtual in GXX_EXTRA_FLAGS (bug #55741)

* configure.ac (GXX_EXTRA_FLAGS): Include -Woverloaded-virtual in the list.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2589,17 +2589,17 @@ AM_CONDITIONAL([AMCOND_INSTALL_BUILD_LOG
 
 dnl Don't add -Wshadow for GCC 4.x.
 case "$GCC_VERSION" in
   *4*) ;;
   *)   GCC_WSHADOW_OPTION=-Wshadow ;;
 esac
 
 GCC_EXTRA_FLAGS="-Wall -W $GCC_WSHADOW_OPTION -Wformat -Wpointer-arith -Wmissing-prototypes -Wstrict-prototypes -Wwrite-strings -Wcast-align -Wcast-qual"
-GXX_EXTRA_FLAGS="-Wall -W $GCC_WSHADOW_OPTION -Wold-style-cast -Wformat -Wpointer-arith -Wwrite-strings -Wcast-align -Wcast-qual"
+GXX_EXTRA_FLAGS="-Wall -W $GCC_WSHADOW_OPTION -Woverloaded-virtual -Wold-style-cast -Wformat -Wpointer-arith -Wwrite-strings -Wcast-align -Wcast-qual"
 
 try_extra_warning_flags=yes
 AC_ARG_ENABLE([extra-warning-flags],
   [AS_HELP_STRING([--disable-extra-warning-flags],
     [don't add -Wall, -W, -Wformat, -Wold-style-cast, and other warning options to CFLAGS and CXXFLAGS])],
   [if test "$enableval" = no; then
      try_extra_warning_flags=no
    fi],
